external/build/deprecation-calendar.js

```diff
@@ -0,0 +1,121 @@
+const {default: fetch} = require('node-fetch');
+const fs = require('fs').promises;
+const path = require('path');
+
+const DEPRECATION_ENDPOINT =
+  'https://chromestatus.com/api/v0/features?q=feature_type=3';
+
+const CHANNELS_ENDPOINT = 'https://chromestatus.com/api/v0/channels';
+
+/**
+ * Fetches all the deprecations from a list of features
+ *
+ * @return {promise} A promise that resolves in an array containing
+ * all the deprecations
+ */
+async function fetchDeprecations() {
+  let deprecations = [];
+  try {
+    const data = await fetch(DEPRECATION_ENDPOINT);
+
+    const deprecationsText = (await data.text()).substring(5); // Remove first 5 chars to have a correctly formatted JSON
```

Comment:
Style nit: move comment above line, but also reuse `let data` for this block instead of defining multiple one-off `const`s.